src/ballet/http/test_live_http_server.c

```diff
@@ -141,7 +141,7 @@ main( int     argc,
                                                         callbacks,
                                                         &state ) );
 
-  FD_TEST( fd_http_server_listen( state.http, 4321U ) );
+  FD_TEST( fd_http_server_listen( state.http, 0, 4321U ) );
```

Comment:
```suggestion
  FD_TEST( fd_http_server_listen( state.http, `INADDR_ANY`, 4321U ) );
```

Reply:
Eh, 0 is pretty common here and avoids a random header.